pkgs/applications/networking/cluster/k3s/1_23/default.nix

```
@@ -174,11 +175,15 @@ let
     vendorSha256 = k3sVendorSha256;
 
     nativeBuildInputs = [ pkg-config ];
-    buildInputs = [ libseccomp ];
+    buildInputs = [ libseccomp sqlite.dev ];
 
     subPackages = [ "cmd/server" ];
     ldflags = versionldflags;
 
+    preBuild = ''
+      export GOFLAGS="-tags=libsqlite3,linux $GOFLAGS"
+    '';
```

Should be able to use `tags` here instead.

```suggestion
    tags = [ "libsqlite3" "linux" ];
```